src/app/Http/Controllers/API/PollController.php

```diff
@@ -120,7 +150,16 @@ public function publish(Request $request, Poll $poll)
      */
     public function results(Request $request, Poll $poll)
     {
+        $res = [];
 
+        foreach ($poll->choices as $choice) {
+            $res[] = [
+                'choice_id' => $choice->id,
+                'votes' => $poll->users()->where('choice_id', '=', $choice->id)->count()
```

Comment:
You should add the belongsToMany relation beetween choice and user to be able to do `$choice->users()->count()` as said before

Reply:
See d9454330e705a1fa1efbbd5cd0d8c6a480c550dd